test/Microsoft.AspNet.Razor.Runtime.Precompilation.Files/TypeDerivingFromITagHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace Microsoft.AspNet.Razor.Runtime.Precompilation
+{
+    public class TypeDerivingFromITagHelper : ITagHelper
```

Comment:
There are other test `TagHelper` types available via `Runtime.Test` why not use those?
Same Q applies to the `FromTagHelper` derivative.